test/sequential/test-watch-mode.mjs

```diff
@@ -791,4 +791,54 @@ process.on('message', (message) => {
       `Completed running ${inspect(file)}`,
     ]);
   });
+
+  it('when multiple `--watch` flags are provided should run as if only one was', async () => {
```

Comment:
I am not sure if it's this PR, but `test-watch-mode` has been extremely flaky in both Jenkins and GitHub actions in the past two days e.g. https://ci.nodejs.org/job/node-test-commit/79645/ https://ci.nodejs.org/job/node-test-commit/79644/ and many others. Please refrain from appending new test cases in existing test files as https://github.com/nodejs/node/blob/main/doc/contributing/writing-tests.md suggests - especially this file because it's already being ignored by Jenkins so you are not going to get any proper coverage by appending test cases  here.

Reply:
I see, thank you very much and sorry for the trouble, I will keep this in mind and create tests in new files whenever possible 🙇